test/libsolidity/semanticTests/functionTypes/function_type_library_internal.sol

```diff
@@ -22,5 +22,7 @@ contract C {
     }
 }
 
+// ====
+// compileViaYul: also
```

Comment:
This is failing because `Utils.sum` does not add the function to the internal dispatch, right?

Reply:
Indeed it's failing for me too. Weird, I was sure all tests were passing when I pushed it yesterday. Maybe I did not actually run all of them after all - I was tweaking the code for that `internalDispatch()` fix in the meantime and also running some specific tests... Good that we have CI to catch stuff like this and do the final clean run.

Anyway, I removed this one and rebased on the current `develop` so it should all be fine now.

Reply:
Oh, and I did not actually answer your question.

Yes, the dispatch is empty and this is most likely the cause of failure:
``` solidity
function dispatch_internal_in_2_out_1(fun , in_0, in_1) -> out_0 {
switch fun

default { invalid() }
}
```